ts/parts/SvgRenderer.ts

```diff
@@ -2528,7 +2528,10 @@ extend((
             duration: pick(duration, 150),
             complete: function (): void {
                 // #3088, assuming we're only using this for tooltips
-                elemWrapper.attr({ y: -9999 });
+                elemWrapper.attr({
+                    y: -9999,
+                    visibility: 'hidden' // #13226
```

Comment:
I would consider to use `.hide()` instead of inline setting visibility.

Reply:
Definitely, thanks for suggestion!